### ISSUE TITLE
Boxed.AspNetCore.TagHelpers.SubresourceIntegrityTagHelper v4.0 with .net core 3.1 

### ISSUE THREAD

**Issue by dotnetshadow** [Dotnet-Boxed/Framework#117]
Firstly thank you for a great project.

### Describe the bug
I've been using your Boxed.AspNetCore.TagHelpers.SubresourceIntegrityTagHelper for a while, but I ran into an issue when I recently updated to .net core 3.1 using your v4.0.0 of the library.

It actually threw an unhandled exception. It seems the wrong path is used 
Version 3.2.0 is OK, version 4.0.0 causes issues for me

...Deployments\Foo\Staging\Foo.Web\js\vendor\jquery.min.js'. doesn't seem to have the wwroot like as follows: Deployments\Foo\Staging\Foo.Web\wwroot\js\vendor\jquery.min.js'.

```
fail: Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware[1]
      An unhandled exception has occurred while executing the request.
System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Programming\Deployments\Foo\Staging\Foo.Web\js\vendor\jquery.min.js'.

   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)

   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)

   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)

   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)

   at System.IO.File.ReadAllBytes(String path)

   at Boxed.AspNetCore.TagHelpers.SubresourceIntegrityTagHelper.ReadAllBytes(String filePath)

   at Boxed.AspNetCore.TagHelpers.SubresourceIntegrityTagHelper.GetSubresourceIntegrityFromContentFile(String contentPath, SubresourceIntegrityHashAlgorithm hashAlgorithms)

   at Boxed.AspNetCore.TagHelpers.SubresourceIntegrityTagHelper.ProcessAsync(TagHelperContext context, TagHelperOutput output)

   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperRunner.<RunAsync>g__Awaited|0_0(Task task, TagHelperExecutionContext executionContext, Int32 i, Int32 count)

   at AspNetCore.Views_Shared__Layout.<>c__DisplayClass78_0.<<ExecuteAsync>b__31>d.MoveNext() in C:\Programming\GitLab-Runner\builds\aacd9b09\0\thedomainpublisher\Foo\src\foo.Web\Views\Shared\_Layout.cshtml:line 410

--- End of stack trace from previous location where exception was thrown ---

   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperExecutionContext.SetOutputContentAsync()

   at AspNetCore.Views_Shared__Layout.<>c__DisplayClass78_0.<<ExecuteAsync>b__1>d.MoveNext() in C:\Programming\GitLab-Runner\builds\aacd9b09\0\thedomainpublisher\foo\src\foo.Web\Views\Shared\_Layout.cshtml:line 396

--- End of stack trace from previous location where exception was thrown ---

   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperExecutionContext.SetOutputContentAsync()
```

### To Reproduce
Unfortunately can't give you a test project just yet

### Expected behaviour
Shouldn't throw an exception

**Comment by RehanSaeed** [comment]
I'll take a PR to fix the issue. Currently trying to get a release ready.

**Comment by dotnetshadow** [comment]
No problem, thanks

I'll do further testing on this, because I think the issue happens when I publish. So there could be something else going on.

**Comment by RehanSaeed** [comment]
@dotnetshadow Any updates?

**Comment by dotnetshadow** [comment]
@RehanSaeed Yeah I tried it the other day and still had the same issue

**Comment by madhousescz** [comment]
There must be change row `this.webHostEnvironment.ContentRootPath,` to `this.webHostEnvironment.WebRootPath,` in SubresourceIntegrityTagHelper.cs on row 248 to include "www" folder. You can add this like project with reference instead of nuget package.

**Comment by RehanSaeed** [comment]
I'll take a PR with a fix.

**Comment by dotnetshadow** [comment]
> 
> 
> There must be change row `this.webHostEnvironment.ContentRootPath,` to `this.webHostEnvironment.WebRootPath,` in SubresourceIntegrityTagHelper.cs on row 248 to include "www" folder. You can add this like project with reference instead of nuget package.
@madhousescz nice, did you encounter the same issue? I would do a pull request but unfortunately due to health won't be able to for a couple of months.

**Comment by RehanSaeed** [comment]
Get well soon!

**Comment by dotnetshadow** [comment]
I'll see if I got time to check if 5.0.0 has fixed this issue, but otherwise maybe PR needs to be made

**Comment by dotnetshadow** [comment]
Hi there, I just tested version 5.0.0 from the nuget package but it caused the same error.

I downloaded the source and referenced it from my project, and it worked. So I'm not sure what's going on if there is a fix in recent days and the nuget package hasn't got any fixes, or for the fact the project is referenced in a certain way. 

I'm going to try creating a nuget package and referencing it from the source to see if it changes anything

**Comment by RehanSaeed** [comment]
Done.

https://github.com/Dotnet-Boxed/Framework/releases/tag/Boxed.AspNetCore.TagHelpers.5.0.1

**Comment by dotnetshadow** [comment]
Hi there,

I just tried the newly published version 5.0.1 but it caused the same error again.
![image](https://user-images.githubusercontent.com/296247/104734722-c9f53a00-5794-11eb-9c44-6784fa70fa13.png)

I'm not sure what is going on, because I made a nuget version myself from source before and I didn't get an error. 

Any thoughts?
